crates/sui-types/src/object.rs

```diff
@@ -651,7 +654,8 @@ impl Object {
     /// we also don't want to serialize the object just to get the size.
     /// This approximation should be good enough for gas metering.
     pub fn object_size_for_gas_metering(&self) -> usize {
-        let meta_data_size = size_of::<Owner>() + size_of::<TransactionDigest>() + size_of::<u64>();
+        let meta_data_size =
+            Owner::SIZE_FOR_GAS_METERING + TransactionDigest::LENGTH + size_of::<u64>();
```

Comment:
Why not use `size_of` for `TransactionDigest`? That shouldn't change, right?

Reply:
It should be the same yes, but I generally wanted to reduce the usage of `size_of` as much as possible. Admittedly, I'm being a bit pedantic.